src/components/ui/sidebar/nav-user.tsx

```diff
@@ -102,7 +102,13 @@ export function FacilityNavUser({
               </DropdownMenuItem>
             </DropdownMenuGroup>
             <DropdownMenuSeparator />
-            <DropdownMenuItem data-cy="user-menu-logout" onClick={signOut}>
+            <DropdownMenuItem
+              data-cy="user-menu-logout"
+              onClick={() => {
+                signOut();
+                navigate("/login");
```

Comment:
why? again, this is just a hack that hides some other problem underneath! 

signOut already does the navigate for you! 

<img width="657" alt="image" src="https://github.com/user-attachments/assets/ef005ae5-dc64-4c96-aba3-ac93ea8d3b22" />